scripts/apps/monitoring/helpers/CustomSortOfGroups.ts

```diff
@@ -0,0 +1,24 @@
+import {get} from 'lodash';
+import {StageGroup} from '../directives/MonitoringGroup';
+
+export const CUSTOM_SORT_SETTING = 'monitoring.customSortOfGroups';
+export const DEFAULT_SORT_FIELD = 'default';
```

Comment:
Doesn't look like a field name. Wouldn't it be better to have `last_updated` or what's the actual default value?

Reply:
This value just means that if it's selected, the sort will match the global sort (so, no custom sort used, which is the default behaviour)

Reply:
How about `const SORT_FIELD = null;` instead? It makes it explicit that the field may not be set and avoids the confusion of using a non-existing field name.

Reply:
But it makes it simpler for the [UI](https://github.com/superdesk/superdesk-client-core/pull/2980/files#diff-4883afa01e282c9d8acc43e8c8a31952R34), because I can just show the fields to sort with this default included, and then the logic is in [the directive](https://github.com/superdesk/superdesk-client-core/pull/2980/files#diff-a65e3fd2ffce9f208245c22a7d5e2639R523)

Reply:
It doesn't look like a lot of work to adjust it to work with `null`. Clean logic is more important in my opinion than saving a few lines of code by using a non-existing field.